### ISSUE TITLE
Move elasticsearch indexing to processors?

### ISSUE THREAD

**Issue by roll** [opentrials/opentrials#41]
I'm not sure how does it work for now - when reindexing happens.

When our system will grow I suppose there should be a long-running `processors` stack updating search index in real time.

@vitorbaptista WDYT?

**Comment by pwalsh** [comment]
I'd say the script that writes the database should also build the index (or, triggered along with)?

**Comment by vitorbaptista** [comment]
Makes sense. I don't think we should bother with updating in real time for now, but, when the time comes,  it'll be easier to reuse the structure you're already creating.
